.env-sample

```diff
@@ -10,6 +10,6 @@ export SET FEATURES="first-feature,second-feature,yet-another-feature"
 export SET GENERAL_RESEARCH_EMAIL=example@nypl.com
 export SET LIB_ANSWERS_EMAIL=example@nypl.com
 export SET SOURCE_EMAIL=example@nypl.com
-export SET LEGACY_CATALOG_BASE_URL=https://[fqdn]/
+export SET LEGACY_BASE_URL=https://[fqdn]/
```

Comment:
Assuming this is a change to accommodate https://github.com/NYPL/discovery-front-end/pull/1684 , I think we may have two different notions of "legacy" in play here. In 1684, I chose "legacyBaseUrl" as a modification of "baseUrl", which is just the base request path for the app. (legacyBaseUrl in that context was meant to indicate a "previous" base path). I have changed the name in 1684 to clarify (it's now "previousBaseUrl"). So I don't think we need the above change, although you're also free to keep it. I'm coming around to the idea that I shouldn't use "legacy" for anything but the old catalog.

Reply:
Oh thanks. I saw that when I glanced at it and made the wrong assumption. I'd like to keep the wording consistent as well. "legacy" here only refers to old catalog